content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -610,8 +610,7 @@ kubectl delete pod myapp myapp-debug
 
 ## Debugging via a shell on the node {#node-shell-session}
 
-If none of these approaches work, you can find the Node on which the Pod is
-running and create a privileged Pod running in the host namespaces. To create
```

Comment:
I think what happened here is that the suggestion to create a privileged pod predates `kubectl debug`. These two statements (1. you can create a privileged pod; 2. you can use `kubectl debug`) are independent even though their proximity implies that they're related.

I think the best way to address this is to move the mention of privileged containers to the "keep in mind" section below, so the flow for the user is:

1. you can create a pod running on the affected node
2. here's how to do this using `kubectl debug`
3. note that this pod isn't privileged and has these limitations
4. if you need a privileged pod you have to create it manually

What do you think?

Reply:
+1 on above idea.

Reply:
@verb Can you provide some texts which I can include? I'd like to see some suggestions hitting up which can be modified if required.

Reply:
@verb if that sounds too onerous, another option is to add your thoughts to the original issue so that folks picking up that work can see what's needed. It's OK if you don't have capacity to write up the text here.

@NitishKumar06 you might be feeling that you're not sure how to move this PR forward without help. If it's easier to, you could consider closing this PR and adding a note to the issue to point out this attempt. That way, a different docs contributor would be able to carry on from where you left off, using the hints from this review to work out what to write.

Reply:
If I'm understanding the technology correctly, the `kubectl debug` command does create a privileged container.

Reply:
@tengqm 

> If I'm understanding the technology correctly, the `kubectl debug` command does create a privileged container.

No, see background in issue https://github.com/kubernetes/website/issues/34879 and the code that creates the debug pod is https://github.com/kubernetes/kubectl/blob/f125772/pkg/cmd/debug/debug.go#L567 ; in contrast, `oc debug` does use `Privileged: true` and `RunAsUser: 0` at https://github.com/openshift/oc/blob/bf53be9/pkg/cli/debug/debug.go#L1011

Reply:
@kgibm Thanks for the clarification.

Reply:
+1